README.md

```diff
@@ -6,6 +6,8 @@
 [![Nightlies](https://github.com/containerd/containerd/workflows/Nightly/badge.svg)](https://github.com/containerd/containerd/actions?query=workflow%3ANightly)
 [![Go Report Card](https://goreportcard.com/badge/github.com/containerd/containerd)](https://goreportcard.com/report/github.com/containerd/containerd)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1271/badge)](https://bestpractices.coreinfrastructure.org/projects/1271)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/containerd/containerd/badge)](https://api.securityscorecards.dev/projects/github.com/containerd/containerd)
```

Comment:
I think it'd be good for us to understand what the scorecard recommendations are before adding a badge.

Reply:
Sure, this would be a good approach. I'll remove it from here then